### PR TITLE
chore: Add statistic stories

### PR DESCRIPTION
**Pull request by alexs-mparticle**
## Instructions

1. PR target branch should be against `main`
2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

## Summary

- Adds Example Stories for Statistics Component based on the examples provided by [Ant Design](https://ant.design/components/statistic)
- Includes visual update to ExampleStory.tsx component by adding a description field and a divider.

## Testing Plan

- [x] Was this tested locally? If not, explain why.
- Visually confirm that components display when running `npm run storybook` and that component controls properly update UI Examples.

## Reference Issue (For mParticle employees only. Ignore if you are an outside contributor)

- Closes https://go.mparticle.com/work/UNI-541

### CODE REVIEW

File: src/components/data-display/Statistic/Statistic.stories.tsx [hunk 1]
```diff
@@ -1,14 +1,242 @@
 import { type Meta, type StoryObj } from '@storybook/react'
-import { Statistic } from 'src/components/data-display/Statistic/Statistic'
+import { Row, Col, Button } from 'antd'
+import { type valueType } from 'antd/es/statistic/utils'
```
Comment (tibuurcio) [inline src/components/data-display/Statistic/Statistic.stories.tsx:3]: Can we import those from src? We tend to not use imports directly from Antd to try and simulate real world usage in stories 🙏🏼

File: src/components/data-display/Statistic/Statistic.stories.tsx [hunk 2]
```diff
@@ -1,14 +1,242 @@
...
+              />
+            </Card>
+          </Col>
+        </Row>
+      </ExampleStory>
+    )
+  },
+}
+
+const deadline = Date.now() + 1000 * 60 * 60 * 24 * 2 + 1000 * 30 // Dayjs is also OK
```
Comment (tibuurcio) [inline src/components/data-display/Statistic/Statistic.stories.tsx:209]: **suggestion** maybe just adding parenthesis to just represent better what are the groupings here? Even if it doesn't need it for precedence.

**Review by tibuurcio — APPROVED** [review]
Thanks for the contribution @alexs-mparticle! Left 2 small suggestions :)

File: src/components/data-display/Statistic/Statistic.stories.tsx (hunk 1 above)
Comment (jared-dickman) [reply src/components/data-display/Statistic/Statistic.stories.tsx:3]: should first export from Statistics 
```export { type AntValueType as valueType }```

**Review by jared-dickman — APPROVED** [review]
looks great!! wondeful cross-squad collaboration on this one alex 🎩

**Pull request merged**